frontend/src/components/QuizCard.vue

```diff
@@ -18,7 +30,8 @@
             </div>
         </div>
         <div class="quiz-details-expand-icon">
-            <p>&#8964;</p>
+            <font-awesome-icon icon="fa-solid fa-chevron-right"
+                               size="l"/>
         </div>
     </div>
 </template>
```

Comment:
This file already has 100 lines. And I think it will grow. As I said, it may be nicer to have a CSS file at some point, but do what you think is best. For the time being, it's fine.

Reply:
I got your point. As soon as a .vue file has more line than this, we will switch to modular CSS for all of them.